### DISCUSSION TITLE
Unify documentation.

### DISCUSSION THREAD

**Discussion by jtagcat** [thinkst/canarytokens#248] [Feature Requests]
Take AWS token for example:

![image](https://user-images.githubusercontent.com/38327267/132961178-08684dcf-690a-4183-90b5-0dd5188e2bc7.png)
(information being hidden, kept from the user; have to guess, and insert example@example.com to show the text)

+

https://docs.canarytokens.org/guide/aws-keys-token.html

***

Proposed:
 - Examples are moved to the generate page, merged with the post-generate text.
 - Collapsed h3 headers are shown, or everything collapsed under 'Info/Docs' or similar. This helps to keep the look simple. Personally, I prefer everything just below the generation form.
 - The result page, with the checkmark, in green, will include minimal information. It may include id-links (jumping to a header / title on the same page, below). The content previously shown is still shown here.
 - The remaining 1 documentation page may be left to the 'docs' (rename to how-to-use). The other is marketing, it can be on the main page as well.
 - The 2 pages can be items at the top of the left bar, #109. I still recommend selecting HTTP by default, showcasing the simplicity (no product or fancy landing page please)

**Comment by roberto-aldera** [comment]
Fixed in UI/docs updates since this was raised
